modules/keycode/keycode.js

```diff
@@ -1,5 +1,6 @@
 /**
  * Enumerates the supported keys.
+ * NOTE: The maps represents actual keys on the keyboard not chars. 
```

Comment:
What do you refer by "key"? In the "NOTE:" I mean hardware key on the keyboard device :)

Although if you are talking about the values in this case they represent the values from robotjs. But this is just enum. If we replace robotjs with something else we can continue using this enum and we'll can make mapping into the wrapper between these values and ones that the new library expect to receive. So I don't think this is a good place to add such a link.

But do you find this file confusing? I would add more comments but I just don't know what to add there.